docker/Dockerfile_i686

```diff
@@ -23,7 +23,7 @@ RUN curl -O -L https://cmake.org/files/v3.9/cmake-3.9.0.tar.gz && \
 	cd .. && \
 	rm -rf cmake-3.9.0*
 
-RUN yum install autoconf automake bzip2 cmake freetype-devel gcc gcc-c++ libtool make mercurial pkgconfig zlib-devel -y && \
+RUN yum install autoconf automake bzip2 cmake freetype-devel gcc gcc-c++ libtool make mercurial pkgconfig openssl-devel zlib-devel -y && \
```

Comment:
CentOS 5's `openssl` is 0.9.8e which is ancient. Is it safe to use?..

Reply:
> CentOS 5's openssl is 0.9.8e which is ancient. Is it safe to use?..

@native-api Wow, I didn't know that will be a problem too. Anyhow, I found [this](https://blacksaildivision.com/how-to-install-openssl-on-centos) simple tutorial to do that. Should I make these changes?

Reply:
I'd go the same way other ffmpeg dependencies are built but that guide looks like a good reference point. Can't say much more, it's your job to see that everything hooks up nicely.

Reply:
@native-api Does this library links system FFmpeg in any case or only use this precompiled FFmpeg? I'm not sure if this patch worked on my machine due to these changes or it used my system FFmpeg during compiling while testing.